templates/_base/master/files/etc-kubernetes-manifests-etcd-member.yaml

```diff
@@ -0,0 +1,128 @@
+filesystem: "root"
+mode: 0644
+path: "/etc/kubernetes/manifests/etcd-member.yaml"
+contents:
+  inline: |
+    apiVersion: v1
+    kind: Pod
+    metadata:
+      name: etcd-member
+      namespace: kube-system
+      labels:
+        k8s-app: etcd
+    spec:
+      initContainers:
+      - name: discovery
+        image: "docker.io/abhinavdahiya/origin-setup-etcd-environment" ## FIXME(abhinav): switch this to official image.
```

Comment:
nit: Is there a location this can be reviewed? Somewhat opaque here. Feel like this should be in final location / reviewed as well to land this. If we merge as-is please open issues tracking that we need to fix this.

Reply:
its in this repo https://github.com/openshift/machine-config-operator/tree/master/cmd/setup-etcd-environment

https://github.com/openshift/release/pull/2036 will allow us to use the official release image.

Reply:
Sorry for the late nit, but can we change the name to something that indicates it’s scoped to either bootstrap or installer or machine?  “installer-bootstrap-etcd” or “machine-master-etcd”?  Would just prefer a bit of scoping

Reply:
@smarterclayton i opened a issue to track the rename https://github.com/openshift/machine-config-operator/issues/152

i will fix this :innocent: